src/EmojiCompletionProvider.ts

```diff
@@ -40,8 +40,7 @@ export default class EmojiCompletionProvider implements vscode.CompletionItemPro
             return this.getMarkupEmojiCompletions(document, replacementSpan);
         }
 
-        return this.getUnicodeEmojiCompletions(document, replacementSpan)
```

Comment:
This change prevented duplicate suggestions, like:

```
:wink:
::wink::
```

It still seems to work with either `:` or `::` prefixes. Is there anything else I'm missing here?